k8s/showcase/vineyard-mars-pytorch/process-data/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM ghcr.io/v6d-io/v6d/vineyard-python-dev
+
+# Install prerequisites
+RUN apt-get update && apt-get install -y \
+    curl --no-install-recommends
+
+RUN curl -LO https://dl.k8s.io/release/v1.24.0/bin/linux/amd64/kubectl && \
+    chmod +x kubectl && \
+    mv kubectl /usr/bin && \
+    pip3 install --no-cache-dir \
+    kubernetes pymars==0.10.0 requests urllib3
+
+WORKDIR /
+
+COPY process-data.py /
+
```

Comment:
Reuse `k8s/test/e2e/Dockerfile`, where a build-arg `PYPI_DEPENDENCIES` is supported.